### ISSUE TITLE
Marketing - Update webmaker.org/app page with updated assets

### ISSUE THREAD

**Issue by thisandagain** [mozilla/webmaker-app#1457]
From https://github.com/MozillaFoundation/plan/issues/418
Depends on #1456

**Comment by thisandagain** [comment]
Assigning to @xmatthewx for now to produce an IA sketch early next week.

**Comment by secretrobotron** [comment]
Btw, this is where the trailer will live, right? i.e. this is the landing page.

**Comment by thisandagain** [comment]
Yup!

**Comment by xmatthewx** [comment]
Here's a pile of example landing pages. Comments and additions always welcome.  https://docs.google.com/presentation/d/1YkUNcI-Qx3Z_BOBubPf6iRaRNGfvM6EK7gtwcx1F_tI/edit?usp=sharing

**Comment by xmatthewx** [comment]
Requirements:
 - Headline & value statement (approx 4-12 words total)
 - Download or "Get it" button (test branded vs not branded)
 - Image of app on phone (photo preferred over illustration)

More:
  - Video
  - 2-3 tiny highlights
   - join _millions_ (on the open web) 
   - free forever
   - description such as: make pages || share stuff || discover & be discovered
  - Share?? (firefox parents who might share a link with their children)

**Comment by secretrobotron** [comment]
> Video

Not a requirement? Would be a link to watch it elsewhere?

**Comment by xmatthewx** [comment]
Reformatted list to fix ambiguity :wink:

**Comment by secretrobotron** [comment]
Hahaha understood. :yellow_heart:

**Comment by HPaulJohnson** [comment]
For period between now and June 19th can we add a box to collect email? As we drive traffic to this page would be good to collect initial interest so we can leverage for launch marketing and content creation.

**Comment by xmatthewx** [comment]
@HPaulJohnson - added email capture to the list above. After we land in the store, we will need to evaluate if download+email compete too much as CTAs

cc @vazquez

**Comment by thisandagain** [comment]
We just need email signup until it's available in the store... then we can switch to "Download".

**Comment by xmatthewx** [comment]
Desktop (Firefox traffic)

## Alpha

![landing-alpha](https://cloud.githubusercontent.com/assets/53411/7420344/a6f1e846-ef49-11e4-920f-8d0e944cc862.png)

## Beta

![landing-beta](https://cloud.githubusercontent.com/assets/53411/7420345/a6f4c610-ef49-11e4-9c03-abf407db2704.png)

**Comment by xmatthewx** [comment]
Mobile

Weather Snap includes an image of a phone on their mobile landing. They do it well, but most benchmarks remove it. I recommend we skip it unless it comes together very easily. 

## Alpha & Beta

![landing-mobile](https://cloud.githubusercontent.com/assets/53411/7420775/365b867e-ef4d-11e4-993c-8d0d4ec8c09b.png)

**Comment by xmatthewx** [comment]
@HPaulJohnson - will find a good quote to replace "this is awesome" and craft improved copy for the rest. @vazquez has already made great progress on UI.

**Comment by HPaulJohnson** [comment]
working on copy for tomorrow. 

another thought after our meeting this morning, along the same lines of whether email is an effective contact method in developing markets, is whether to include whatsapp sharing option along with FB & twitter?

**Comment by vazquez** [comment]
This is where I got to today with UI for the site. :cow: :star: 

##### Landing Page with Blue Solid Treatment
![app landing page blue solid](https://cloud.githubusercontent.com/assets/1119821/7421342/b5ad679a-ef51-11e4-950c-e7cab334bcec.png)

##### Landing Page with Photo Treatment
![app landing page photo](https://cloud.githubusercontent.com/assets/1119821/7421344/b5b38f44-ef51-11e4-81fa-935b524ba431.png)

##### Landing Page with Gradient Treatment
![app landing page purple gradient](https://cloud.githubusercontent.com/assets/1119821/7421343/b5b19a2c-ef51-11e4-8bce-2b6a43a7f81c.png)

:boom:

**Comment by thisandagain** [comment]
O_O Dang folks! Some serious progress this afternoon. :boom: 

*Really* digging the direction here ... only major concern is that I feel like the product isn't quite being showcased enough. The angle of the phone cuts down on the amount of surface area for the product quite a bit. Wondering if this could be resolved by a different angle or possibly even multiple devices?

**Comment by thisandagain** [comment]
One other detail, for Beta we should add the Google Play logo somewhere in or near the "Get the App" button. User's are pretty savvy about recognizing that icon and I've seen some pretty major positive results by including it with other products.

**Comment by xmatthewx** [comment]
@thisandagain - we were thinking VR 360˚ phone rotation with playable app embedded on it. Would that work?

**Comment by thisandagain** [comment]
LOL :laughing: Perfect.... only if it works on Oculus Rift.

**Comment by xmatthewx** [comment]
Agreed: a flatter phone angle would help. 

Android: I figured we could a/b test branded vs not branded "Get App" button. Benchmarks seem truly divided. I'd also guess that Play branding will help us get off the ground, but seems easy enough to test while we run tests on the copy.

**Comment by thisandagain** [comment]
Sounds good. Great thing to test with optimizely. :+1:

**Comment by xmatthewx** [comment]
@thisandagain - Forgot to mention. Would be nice to include a minimal animation on the phone that zooms between map and page view. Could be JS rather than GIF. Simple, just two states, with nice pause in between. 

@vazquez - I'd like the share options to be muted a bit. Maybe aligned with the baseline of Webmaker? I'm drawn to the photo. Maybe we can test this against the gradient. Beautiful work. 

Lastly, when this design settles, let's remember to 
 - [ ] add a variation as the cover to the intro deck.

**Comment by thisandagain** [comment]
Sounds good. Doing it with a CSS transition should help with any performance and page weight issues. +1 on the gradient iteration. I'm worried about page weight (particularly with our target markets) with the photo.

**Comment by vazquez** [comment]
@xmatthewx - I can't find share buttons on Webmaker.org. Could you point me in the right direction? By "muted" do you mean lowering opacity/size?

**Comment by xmatthewx** [comment]
@vazquez - 

 - [ ] **Share** - Adjust opacity and/or size as you see fit. Buttons draw too much attention away from the app and CTA. Sharing on the webmaker blog and in thimble is suboptimal. No need to reference it.
 - [ ] **Heading** - Match secondary heading on gradient version to the other two. "Capture, collect..."
 - [ ] **Quote** - Replace "Join Millions" with a placeholder quote: "So easy to share my own pages. Incredible!" -- Ricardo Vazquez (yep, you said it)  
 - [ ] **Angle** - Adjust the phone angle and we'll be in great shape. 
 - [ ] **Content** - We can tackle project content for this and the video simultaneously.

**Comment by vazquez** [comment]
##### Evolution 

![app landing page photo](https://cloud.githubusercontent.com/assets/1119821/7433583/ef11aea6-f000-11e4-8aa4-a102b4ff2b53.png)

---

![app landing page purple gradient](https://cloud.githubusercontent.com/assets/1119821/7433585/ef237366-f000-11e4-9ec7-5932f7da0b9e.png)

---

##### Beta Version
![app landing page purple beta](https://cloud.githubusercontent.com/assets/1119821/7433584/ef192564-f000-11e4-8090-cc6ca17ed255.png)

**Comment by davidascher** [comment]
whoa

**Comment by Pomax** [comment]
These are amazingly clean and lovely. My personal favourite is the nature bg, since it hints at a toucan.

Question about the share/tweet buttons: will these be lazy loading (to prevent tracking) or just direct buttons?

**Comment by LauraReynal** [comment]
I don't know if I have been on that website too frquently, but as much as I love the purple tinker mode, it reminds me a lot of http://internet.org/ ... which we might want to consider for branding.

**Comment by thisandagain** [comment]
Good point @LauraReynal . Specifically they use a blue to violet gradient in their Android app (used in the navigation bar). Thoughts @HPaulJohnson ?

**Comment by xmatthewx** [comment]
> nature bg

Probably too big an asset without squeezing in noise. But, it's hard to resist the impact of a photo background. 

> share/tweet buttons

A nice to have feature. Simple links should be sufficient. 

> internet.org

Ugh. We could shift gradient right a bit to make it more blue-to-purple. They're squarely purple-to-pink.

**Comment by Pomax** [comment]
oh dear. let' steer clear of internet.org associations. Aside from the facebook initiative, news reports from India are hinting at it being a sinking ship atm...

**Comment by davidascher** [comment]
We should explore a low pass filtered photo asset, esp. given how jpeg
works it might compress very very well, and we don't want high detail
anyway.

@pomax: internet.org is all of those things and also strategically
complicated wrt Webmaker. Thanks Laura for noticing!

**Comment by xmatthewx** [comment]
If we _can_ use a compressed photo, problem solved. @vazquez share a link to the image.

Either way, we should reclaim purple gradients for posterity! :grapes: :smiling_imp:

**Comment by vazquez** [comment]
https://dl.dropboxusercontent.com/u/43783651/Mozilla/Android.zip

**Comment by vazquez** [comment]
Agree with @xmatthewx - :purple_heart:

**Comment by HPaulJohnson** [comment]
These are looking awesome. I agree with all the comments that we should be as distinct as possible from internet.org. Appreciate @LauraReynal flagging that.

**Comment by vazquez** [comment]
Adding a teal gradient for consideration.

![app landing page teal gradient](https://cloud.githubusercontent.com/assets/1119821/7436763/911e510a-f01e-11e4-964f-687b270fb835.png)

**Comment by xmatthewx** [comment]
@HPaulJohnson @thisandagain - what do we need in the footer? Link to legal and privacy? Anything else?

**Comment by thisandagain** [comment]
- Legal (TOS)
- Privacy
- Contact

**Comment by vazquez** [comment]
#### Updating UI

Adding Footer:

![app landing page photo](https://cloud.githubusercontent.com/assets/1119821/7478929/0644b1b2-f313-11e4-9d78-a8cc48347ffb.png)

---

![app landing page teal gradient](https://cloud.githubusercontent.com/assets/1119821/7478931/06476e34-f313-11e4-89fe-f8fd2a544129.png)

---

##### Beta version of Landing Page

![app landing page teal beta](https://cloud.githubusercontent.com/assets/1119821/7479130/758f6cdc-f314-11e4-8595-4ea0277c3af6.png)

**Comment by thisandagain** [comment]
Perfect. Thanks @vazquez ! :smile: 

@HPaulJohnson where would you like us to record / capture these email addresses? /cc @adamlofting

**Comment by HPaulJohnson** [comment]
@thisandagain is BSD an option? if not, what are the options? @adamlofting is on PTO or else I would defer to him on this. My main requirement is to be able to contact them around the launch, ideally through a bulk emailer.

**Comment by xmatthewx** [comment]
Is it useful to capture this in Webmaker's DB so we can track the origin of users later? Too complicated? 

 - [ ] @vazquez add checkbox icon and legalese when you have other content for comp

**Comment by thisandagain** [comment]
@HPaulJohnson Probably. I'll try to dig out some details.

@xmatthewx Not particularly, also a way bigger lift as we'd need to create a new table and API endpoint just for this.

**Comment by vazquez** [comment]
#### Updating UI

Adding checkmark with Legal terms.

![app landing page teal gradient](https://cloud.githubusercontent.com/assets/1119821/7499378/03c5ec8e-f3dc-11e4-8170-78a4a9ea28d2.png)

---

##### Beta version of Landing Page

Beta version remains unchanged.

![app landing page teal beta](https://cloud.githubusercontent.com/assets/1119821/7499302/8e3623ee-f3db-11e4-9734-031e46fd636b.png)

**Comment by vazquez** [comment]
Sketch file! :santa: https://www.dropbox.com/s/cwoegt612voxydk/App%20Landing%20Page.sketch?dl=0

**Comment by HPaulJohnson** [comment]
This is looking awesome. A few notes about copy: 

* For headline I think we should have a set of 6-12 different versions by the end of the week. I assume we are setting this up with Optimizely so we can test? If that's the case, working version we have here is fine. 
* For quote, almost all of the press to-date on Webmaker has been around beta version from MWC and our digital inclusion work. It's been tough to find even a short pull quote that works well. I've added a few ideas in this etherpad (and feel free to browse the press hits yourself to find something better) but we may just want to use a user quote for now and update later when we start to do a bigger press push around launch. 
* Copy for 'Free Forever' is also on etherpad. It's the same copy from existing site and if it needs to be shortened let me know. 
* Copy for a third section ('Make Pages' in earlier comps) also included but I'm fine with dropping that section altogether based on latest design. 

https://etherpad.mozilla.org/comingsoonpage

**Comment by LauraReynal** [comment]
@HPaulJohnson is the quote necessarly from a newspaper or could it be from a user ?

**Comment by HPaulJohnson** [comment]
User works fine at this point. Eventually if we get some great press pickup we can update as it adds a different level of 3rd party validation. But if you have great user quotes let's use them now.

**Comment by xmatthewx** [comment]
@LauraReynal - a user quote could be about the app or about the desire to share, connect, etc.

**Comment by LauraReynal** [comment]
Added quotes in the pad from previous reports + Chicago new ones 

Line 50 - we could cut the parenthesis 

Let me know if okay :rabbit:

**Comment by LauraReynal** [comment]
@HPaulJohnson Here is another quote that I just found you might like 

"being able to create things is amazing, because a lots of time you don’t have the opportunity to create something, have ownership and people be a part of it" Chicago, USA

**Comment by thisandagain** [comment]
Tracking in https://github.com/mozilla/webmaker-desktop/issues/30

**Comment by adamlofting** [comment]
Belated response re where to save the emails: Yes to using BSD. 

I've dropped some reference material in to the corresponding ticket https://github.com/mozilla/webmaker-desktop/issues/33
